irods_catalog_consumer/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:20.04
```

Comment:
Consider changing this to at least 22.04. Don't have to.

Reply:
Tried it. Required more changes - so will separate that upgrade.